server/sonar-server/src/main/java/org/sonar/server/ws/WebServiceEngine.java

```diff
@@ -125,6 +129,14 @@ public void execute(Request request, Response response, String controllerPath, S
     return action;
   }
 
+  private static void checkFormatSuffix(@Nullable String suffix) {
+    if (StringUtils.isBlank(suffix)) {
+      return;
+    }
+
+    checkArgument(SUPPORTED_MEDIA_TYPES_BY_URL_SUFFIX.get(suffix.toLowerCase(Locale.ENGLISH)) != null, "Unknown action suffix: %s", suffix);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'suffix' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)